tests/unit/core/reducers/test_addons.js

```diff
@@ -78,61 +69,58 @@ describe(__filename, () => {
   });
 
   it('stores all add-ons, indexed by id', () => {
-    const addonResults = [
-      { ...fakeAddon, slug: 'first-slug', id: 123 },
-      { ...fakeAddon, slug: 'second-slug', id: 456 },
-    ];
-    const state = addons(undefined, loadAddonResults({ addons: addonResults }));
+    const addon1 = { ...fakeAddon, slug: 'first-slug', id: 123 };
+    const addon2 = { ...fakeAddon, slug: 'second-slug', id: 456 };
+
+    let state = addons(undefined, loadAddon({ addon: addon1 }));
+    state = addons(state, loadAddon({ addon: addon2 }));
```

Comment:
I decided to do this because some of these test cases cover more than one behavior at a time.